pkg/bme680_driver/contrib/bme680_hal.c

```diff
@@ -0,0 +1,87 @@
+
+#include <stdio.h>
+#include <string.h>
+
+#include "bme680_hal.h"
+
+#ifdef MODULE_PERIPH_I2C
+#include "periph/i2c.h"
+#endif
+#ifdef MODULE_PERIPH_SPI
+#include "periph/spi.h"
+#endif
+
+#include "xtimer.h"
+
+#ifndef BME680_SPI_SPEED
+#define BME680_SPI_SPEED    (SPI_CLK_1MHZ)
+#endif /* BME680_SPI_SPEED */
+
+#ifndef BME680_SPI_MODE
+#define BME680_SPI_MODE     (SPI_MODE_0)
+#endif /* BME680_SPI_MODE */
+
+#ifndef BME680_NSS_PIN
```

Comment:
BTW, why is it called `BME680_NSS_PIN`, the datasheet call it `CSB`?

Reply:
Old habit I guess, I can switch it to the datasheet name if you prefer